src/functionalTest/resources/scenarios/RIA-5912-submit-application-uploadB1Document.json

```diff
@@ -0,0 +1,61 @@
+{
+  "description": "Submit the application with a B1 form (pre-submit REST call)",
+  "request": {
+    "uri": "/bail/ccdAboutToSubmit",
+    "credentials": "AdminOfficer",
+    "input": {
+      "id": 1111,
+      "eventId": "submitApplication",
+      "state": "applicationStarted",
+      "caseData": {
+        "template": "minimal-application-started.json",
+        "replacements": {
+          "uploadB1FormDocs":
+          [
+            {
+              "id": "3",
+              "value": {
+                "document": {
+                  "document_url": "http:\/\/dm-store:8080\/documents\/aaaaaaaa-bbbb-1ccc-8ddd-eeeeeeeeeeee",
+                  "document_binary_url": "http://dm-store:8080/documents/aaaaaaaa-bbbb-1ccc-8ddd-eeeeeeeeeeee\/binary",
+                  "document_filename": "B1.PDF"
+                },
+                "description":"test b1 description"
+              }
+            }
+          ]
+        }
+      }
+    }
+  },
+  "expectation": {
+    "status": 200,
+    "errors": [],
+    "caseData": {
+      "template": "minimal-application-started.json",
+      "replacements": {
+        "uploadB1FormDocs":
```

Comment:
Same as above, on the expectation, this should be applicantDocumentsWithMetadata instead of uploadB1FormDocs.

REASON: By the end of the UploadB1DocumentHandler class, the document you uploaded into uploadB1FormDocs will be added to the applicantDocumentsWithMetadata collection.